modulefiles/mpi/compilerName/compilerVersion/mpiName/mpiVersion/met/met.lua

```diff
@@ -0,0 +1,34 @@
+help([[
+]])
+
+local pkgName    = myModuleName()
+local pkgVersion = myModuleVersion()
+local pkgNameVer = myModuleFullName()
+
+local hierA        = hierarchyA(pkgNameVer,2)
+local mpiNameVer   = hierA[1]
+local compNameVer  = hierA[2]
+local mpiNameVerD  = mpiNameVer:gsub("/","-")
+local compNameVerD = compNameVer:gsub("/","-")
+
+conflict(pkgName)
+
+always_load("ips")
+always_load("python/3.6.3")
+prereq("ips")
+prereq("python/3.6.3")
```

Comment:
Specific to WCOSS.  Please remove.

Reply:
I think the requisites for met are more than these.  They are probably incorrectly required in metplus modulefile.